tests/stl/list_test.cpp

```diff
@@ -0,0 +1,54 @@
+#include "gtest/gtest.h"
```

Comment:
@Ambyjkl define the list template beforehand in your test. Check @Mogball PR for HashMap and see how he is doing that. Doing this will let you see how much code is covered by your test

Reply:
Basically for each template specialization you use, declare that class at the top of the file like

```c++
namespace wlp {
    template class List<int>;
    template class List<char*>;
    template class List<StaticString<8>>;
...
}
```

Reply:
OK so I just added this to the top of the file:
```
namespace wlp {

    template class List<uint16_t>;

}
```

And it seems to break the test for some reason

Reply:
I'm getting a code coverage of 100% even without that, why do we need to do this? Btw wmake coverage is broken since lcov isn't removing a lot of files. I tried playing around with lcov --remove but it won't listen to me. I did manage to make it forget about '/usr/*', so I guess it needs absolute paths? I'm pushing my changes, have a look.

Reply:
You are getting 100% coverage because the coverage check doesn't see all the functions. You need to declare the class so that the coverage check can find all functions.

Reply:
But do you know why adding it broke the tests? I get an issue with List.at(), even though it passed the assertion earlier.

Reply:
Actually I have no idea. You'd have to debug a bit.